packages/app/.prettierrc.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  ...require("../../.prettierrc.js")
```

Comment:
Wouldnt it be better to use a global prettier config so that all of the projects are unified?

Reply:
yes but that quickly gets messy, for editor plugins/pre-commit-hooks/etc it really makes life a lot easier if you have a config in your cwd